moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -513,9 +538,75 @@ KinematicState Servo::getNextJointState(const ServoInput& command)
     }
   }
 
+  // remove old commands
+  cur_time = node_->now();
+  while (!committed_commands_.empty() &&
+         committed_commands_.front().time < (cur_time - rclcpp::Duration::from_seconds(2 * servo_params_.latency)))
```

Comment:
You're doing the `2 * servo_params_.latency` calculation in here, though `dt` is defined below. Maybe just define that earlier on and keep reusing the variable?